x-pack/plugins/synthetics/server/legacy_uptime/lib/requests/get_monitor.ts

```diff
@@ -38,7 +38,8 @@ export const getSyntheticsMonitor = async ({
           namespace: encryptedMonitor.namespaces?.[0],
         }
       );
-    return normalizeSecrets(decryptedMonitor);
+    const { attributes } = normalizeSecrets(decryptedMonitor);
+    return attributes;
```

Comment:
I see that this function is utilized by our get single monitor apis, so this answers my question above.